DigitalLibraryofTennessee/dltn_ingestion3_config_generator
# Add ten new sets for December DPLA ingest.

**GitHub Issue**: [283](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/283)

## What does this Pull Request do?

Adds new sets for the December 2022 DPLA ingest. All sets have been tested for requirements.

## How should this be tested?

- Ensure that the sets match the Data Set name in Repox
- Check that sets listed in GitHub issue are present in config file
